pkg/controller/combo_controller.go

```diff
@@ -0,0 +1,45 @@
+package controller
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/operator-framework/combo/api/v1alpha1"
+)
+
+type combinationController struct {
+	client.Client
+
+	log logr.Logger
```

Comment:
```suggestion
	client.Client
	log logr.Logger
```

Reply:
This was not addressed.

Reply:
Oops, I guess it got lost in some rebasing I was doing a bit back. Fixed now!